### PR TITLE
minikube pause command, description fix

### LINKED ISSUE #14773: Docs: stop vs pause
### What Happened?

I see that minikube containers in minikube consume ressources, so that my laptop fan makes noise.

At the moment I don't need minikube.

I look at 

* https://minikube.sigs.k8s.io/docs/commands/stop/
* https://minikube.sigs.k8s.io/docs/commands/pause/

I have no clue what the difference between both commands.

It would be great, if the docs would explain the difference between both commands.

### Attach the log file

.

### Operating System

Ubuntu

### Driver

Docker

### PR DESCRIPTION
**Pull request by Aryan-Deshpande**
fixes #14773

### CODE REVIEW

**Comment by Aryan-Deshpande** [comment]
Updated branch with changes, would like to know if I'm missing anything to be added.

File: site/content/en/docs/commands/pause.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ description: >
 
 ## minikube pause
 
-pause Kubernetes
+Pauses a running local Kubernetes cluster
```
Comment (medyagh) [inline site/content/en/docs/commands/pause.md:10]: I think it is already implied that this is a local Kubernetes,
How about, 
“Pauses the api-server for the active Kubernetes cluster”

File: site/content/en/docs/commands/pause.md [hunk 2]
```diff
@@ -7,11 +7,12 @@ description: >
...
 
 ### Synopsis
 
-pause Kubernetes
+Pauses local Kubernetes cluster and free up CPU resources
```
Comment (medyagh) [inline site/content/en/docs/commands/pause.md:14]: Let’s add this, to the description
“Without pausing pods/apps inside the cluster.”

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/commands/pause.md (hunk 2 above)
Comment (Aryan-Deshpande) [reply site/content/en/docs/commands/pause.md:14]: As in "Pauses local Kubernetes cluster and free up CPU resources, without pausing pods/apps inside the cluster" ?

**Comment by klaases** [comment]
Hi @Aryan-Deshpande, did you have a chance to review Ander's above comments for your PR?

**Comment by Aryan-Deshpande** [comment]
hey, @klaases I did review his comments on my PR :). I researched quite a bit and concluded that the minikube stop only halts the container that the minikube runs on. He also shed light on how the use of "minikube pause", pauses the Kubernetes cluster processes and not the docker container that minikube runs on. Right now there is no functionality to pause a VM/container, as a VM either shuts down completely or the whole VM has to be booted again. Please let me know if I was wrong, I would love to learn!

File: site/content/en/docs/commands/pause.md (hunk 2 above)
Comment (Aryan-Deshpande) [reply site/content/en/docs/commands/pause.md:14]: I believe "minikube pause", pauses all the applications running in k8s cluster and the k8s cluster itself. Please correct me if I am wrong.

**Comment by Aryan-Deshpande** [comment]
Would like to get the changes reviewed !

**Comment by Aryan-Deshpande** [comment]
hey @medyagh any other changes to be made ?.

**Pull request closed without merging**